### PR TITLE
Added documentation for BalancedResourceAllocation priority function

### PR DESCRIPTION
**Pull request by abhgupta**
@adellape PTAL

### CODE REVIEW

File: admin_guide/scheduler.adoc [hunk 1]
```diff
@@ -74,6 +74,10 @@ These priority functions do not take any configuration parameters or inputs from
 ----
 {"name" : "LeastRequestedPriority", "weight" : 1}
 ----
+**_BalancedResourceAllocation_** favors nodes with balanced resource usage rate.  It calculates the difference between the comsumed CPU and memory as a fraction of capacity, and prioritizes the nodes based on how close the two metrics are to each other.  This should always be used together with _LeastRequestedPriority_.
```
Comment (adellape) [inline admin_guide/scheduler.adoc]: s/comsumed/consumed/

**Comment by adellape** [comment]
@abhgupta Thanks, just the one comment inline.

**Comment by abhgupta** [comment]
@adellape Fixed.

**Comment by adellape** [comment]
@abhgupta Thanks, merging.

**Pull request merged**